app/app.component.js

```diff
@@ -0,0 +1,10 @@
+(function(app) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Move the invocation into the parens that contain the function.](https://www.codacy.com/app/garethjackson/www-heptonstall-info/file/2734059926/issues/source?bid=3331202&fileBranchId=3351007#l1)